lib/main.coffee

```diff
@@ -71,8 +166,14 @@ module.exports =
             col = parseInt(match[2]) or 0
             toReturn.push({
               type: if match[4] is 'E' then 'Error' else 'Warning'
-              text: match[3] + '- ' + match[5]
+              text: match[3] + ' — ' + match[5]
```

Comment:
Any particular reason you switched this to an emdash? `flake8` on the command line doesn't do this, and it makes the messages take up more space.